security/middleware.py

```diff
@@ -137,30 +168,36 @@ def load_setting(self, setting, value):
             return
         value = value.lower()
         if value not in XssProtectMiddleware.OPTIONS.keys():
-            raise ImproperlyConfigured(XssProtectMiddleware.__name__ + " invalid option for XSS_PROTECT.")
+            raise ImproperlyConfigured(
+                XssProtectMiddleware.__name__ +
+                " invalid option for XSS_PROTECT.",
+            )
+
         self.option = value
 
     def process_response(self, request, response):
         """
         Add X-XSS-Protection to the reponse header.
         """
-        response['X-XSS-Protection'] = XssProtectMiddleware.OPTIONS[self.option]
+        response['X-XSS-Protection'] = \
```

Comment:
I find the benefits from being 'totally' PEP8 compliant, and therefore being able to add it as a test stage that can fail (as I have done in the tox config) outweigh the slight annoyances in cases like this.

Happy to re-format to use parentheses instead, however I think sticking to 80 chars is important, ever more so given the availability of widescreen monitors where multiple files can be viewed simultaneously.